IntentsExtension/Intents.intentdefinition

```diff
@@ -824,6 +824,69 @@
 				</dict>
 			</array>
 		</dict>
+		<dict>
+			<key>INEnumDisplayName</key>
+			<string>Comparison Type</string>
```

Comment:
<img width="1361" src="https://user-images.githubusercontent.com/137/147799848-d6bf6111-1f2f-4878-9e3f-1a8b0031acac.png">

Reply:
These initial comparison options are in a rough draft state. It'd be nice if the natural language could read like:

> Get items in both <List 1> and <List 2>
>
> Get items in <List 1> but not in <List 2>

But I'm not quite sure how we can achieve this if everything is packed into a single action.

Reply:
You can achieve that by using separate variables for "List 1" and "List 2" for each comparison type. See the "EditURL" action for an example.

Reply:
Nice, that's a pretty neat trick

<img width="506" src="https://user-images.githubusercontent.com/137/147830579-d777fae2-122c-41e1-881d-b81ceea82927.png">
.